src/test_c2s_srv.c

```diff
@@ -472,10 +472,9 @@ int test_c2s(Connection *ctl, tcp_stat_agent *agent, TestOptions *testOptions,
   create_client_logdir((struct sockaddr *) &cli_addr[0], clilen,
                        options->c2s_logname, sizeof(options->c2s_logname),
                        namesuffix, sizeof(namesuffix));
-  sleep(RACE_CONDITION_WAIT_TIME);
+  mysleep(RACE_CONDITION_WAIT_TIME);
```

Comment:
My thinking for the change to sleep comes from the man pages for sleep & alarm: "sleep(3) may be implemented using SIGALRM; mixing calls to alarm() and sleep(3) is a bad idea."